etl/steps/data/garden/regions/2023-01-01/regions.yml

```diff
@@ -1350,6 +1350,7 @@
     - "Korea, Rep."
     - "Korea, Republic Of"
     - "Republic of Korea"
+    - "Korea"
```

Comment:
I'm not sure this is a good alias. "Korea" does not mean "South Korea", even if for this particular dataset it may mean that.